### PR TITLE
Commit message was too long, see desc 👇

### PR DESCRIPTION
**Pull request by Um-Brehon**
Created the introduction component for the home page with a link to a more detailed introduction page also created here. Created the links on the App.jsx file.

### CODE REVIEW

File: src/components/About/style.jsx [hunk 1]
```diff
@@ -0,0 +1,56 @@
+import styled from 'styled-components';
+
+const SAbout = styled.body`
```
Comment (KeerhemFR) [inline src/components/About/style.jsx]: body is already in index.html, should use something else

File: src/components/Home/Introduction/style.jsx [hunk 2]
```diff
@@ -0,0 +1,90 @@
+import styled from 'styled-components';
+
+const SIntroduction = styled.section`
+  .introduction {
+    width: 100vw;
+    height: auto;
+    display: flex;
+    flex-flow: column;
```
Comment (KeerhemFR) [inline src/components/Home/Introduction/style.jsx]: why flex-flow if only using direction? add a wrap or no-wrap value in the flex-flow, or change it to a simple flex-direction

File: src/components/Home/Introduction/style.jsx [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+  .aljPicture {
+    width: 100%;
+  }
+  .aljAchievementsList {
+    background-color: white;
+    width: 90vw;
+    padding: 20px;
+    font-size: 18px;
+    display: flex;
+    flex-flow: column;
```
Comment (KeerhemFR) [inline src/components/Home/Introduction/style.jsx]: same

File: src/components/Home/Introduction/style.jsx [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+    justify-content: center;
+    align-items: center;
+  }
+
+  @media only screen and (min-width: 700px) {
+    .introduction {
+      width: 100vw;
+      color: black;
+      justify-content: flex-start;
+      flex-flow: row;
```
Comment (KeerhemFR) [inline src/components/Home/Introduction/style.jsx]: same

File: src/components/Home/index.jsx [hunk 5]
```diff
@@ -0,0 +1,11 @@
+import Introduction from './Introduction';
+
+function Home() {
+  return (
+    <body>
+      <Introduction />
+    </body>
```
Comment (KeerhemFR) [inline src/components/Home/index.jsx]: body already in index.html, should be something else

File: src/components/Home/style.jsx [hunk 6]
```diff
@@ -0,0 +1,15 @@
+import styled from 'styled-components';
+
+const SHome = styled.body`
+  body {
```
Comment (KeerhemFR) [inline src/components/Home/style.jsx]: same

File: src/components/Home/style.jsx [hunk 7]
```diff
@@ -0,0 +1,15 @@
...
+    width: 80%;
+  }
+  @media only screen and (min-width: 700px) {
+    body {
```
Comment (KeerhemFR) [inline src/components/Home/style.jsx]: same

**Review by KeerhemFR — APPROVED** [review]
Oups, approved by a missclick

**Review by KeerhemFR — CHANGES_REQUESTED** [review]
Some fix are required

**Review by KeerhemFR — APPROVED** [review]
See no evil

**Review by Hacene010 — APPROVED** [review]

**Pull request merged**